OpenRA.Game/ModMetadata.cs

```diff
@@ -23,6 +23,7 @@ public class ModMetadata
 		public string Description;
 		public string Version;
 		public string Author;
+		public bool Hidden;
```

Comment:
Not a fan of camel case there. Also we don't have several hide modes to differentiate.